.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2.1
 jobs:
   build:
     docker:
-      - image: 218546966473.dkr.ecr.us-east-1.amazonaws.com/circle-ci:tap-tester
+      - image: 218546966473.dkr.ecr.us-east-1.amazonaws.com/tap-tester:4.latest
```

Comment:
@dbshah1212 Is it intentional for changing this version?

Reply:
Offline discussed with Dhruvin.. It is intentional

Reply:
circle-ci:tap-tester-v4 -> Please change the version

Reply:
circle-ci:tap-tester-v4 -> Please change the version